### PR TITLE
7726 - adding a way to prevent the feedback panel being shown on certain pages

### PR DESCRIPTION
**Pull request by jongilbraith**
We only have a handful of pages that need this right now and it's not expected to change frequently, so went with simply hard coding a list of slugs into a constant.

---
This change is [Reviewable](https://reviewable.io/reviews/moneyadviceservice/frontend/1594)

### CODE REVIEW

File: app/views/shared/_feedback.html.erb [hunk 1]
```diff
@@ -1,15 +1,9 @@
...
-<% end %>
+<section class="panel feedback-panel">
+  <h2 class="feedback-panel__title"><%= t('feedback_panel.title') %></h2>
+  <p><%= t('feedback_panel.greeting') %></p>
+  <ul>
+    <li><%= link_to t('feedback_panel.article_feedback_link_text'), new_article_feedback_path(article.object.id), class: 't-feedback-panel__article-feedback-link' %></li>
+    <li><%= link_to t('feedback_panel.technical_feedback_link_text'), new_technical_feedback_path, class: 't-feedback-panel__technical-feedback-link' %></li>
+    <li><%= link_to t('feedback_panel.advice_link_text'), advice_path, class: 't-feedback-panel__advice-link' %></li>
+  </ul>
+</section>
```
Comment (jongilbraith) [inline app/views/shared/_feedback.html.erb]: @tomas-stefano yes, that's a different type of feedback which was under another feature toggle that's no longer used, so I cleaned it up and removed the it to save other people getting confused (like me).

Comment (jongilbraith) [inline app/views/shared/_feedback.html.erb]: @tomas-stefano I'm going to remove this commit from this branch and create a separate tech debt task to look for and clean up any forgotten feature toggles. I want to check this out a bit more before making the changes.

Comment (tomas-stefano) [inline app/views/shared/_feedback.html.erb]: Sounds perfect!

**Comment by tomas-stefano** [comment]
👍

**Comment by rnalexander** [comment]
:+1: :shipit:

**Pull request merged**